CONTRIBUTING.md

```diff
@@ -19,23 +19,23 @@ Before you start a load of work, please note that all Pull Requests (apart from
+This document serves as guide for contributing code changes to `dbt-snowplow-unified`. It is not intended as a guide for using `dbt-snowplow-unified`, and some pieces assume a level of familiarity with Python development (virtualenvs, `pip`, etc) and dbt package development. Specific code snippets in this guide assume you are using macOS or Linux and are comfortable with the command line.
 
 ### Notes
 
-- **CLA:** If this is your first time contributing you will be asked to sign the Individual Contributor License Agreement. If you would prefer to read this in advance of submitting your Pull Request you can find it [here](https://docs.google.com/forms/d/e/1FAIpQLSd89YTDQ1XpTZbj3LpOkquV_h1Y8k9ay3iFbJsZsJrz18I23Q/viewform). If you are unable to sign the CLA, the `dbt-snowplow-*` maintainers will unfortunately be unable to merge any of your Pull Requests. We welcome you to participate in discussions, open issues, and comment on existing ones.
+- **CLA:** If this is your first time contributing you will be asked to sign the Individual Contributor License Agreement. If you would prefer to read this in advance of submitting your Pull Request you can find it [here](https://docs.google.com/forms/d/e/1FAIpQLSd89YTDQ1XpTZbj3LpOkquV_h1Y8k9ay3iFbJsZsJrz18I23Q/viewform). If you are unable to sign the CLA, the `dbt-snowplow-unified` maintainers will unfortunately be unable to merge any of your Pull Requests. We welcome you to participate in discussions, open issues, and comment on existing ones.
 - **Branches:** All Pull Requests from community contributors should target the `main` branch (default) and the maintainers will create the appropriate branch to merge this into. Please let us know if you believe your changes are a breaking change or could be done as part of a patch release, if you are unsure that's fine just make that clear in your Pull Request.
 - **Documentation:** The majority of the documentation for our dbt packages is in the core [Snowplow Docs](https://github.com/snowplow/documentation) and as such a Pull Request will need to be raised there to update any docs related to your change. Things such as the deployed dbt site are taken care of automatically.
 
 ## Getting the code
 
 ### Installing git
 
-You will need `git` in order to download and modify the `dbt-snowplow-*` source code. On macOS, the best way to download git is to just install [Xcode](https://developer.apple.com/support/xcode/).
+You will need `git` in order to download and modify the `dbt-unified` source code. On macOS, the best way to download git is to just install [Xcode](https://developer.apple.com/support/xcode/).
```

Comment:
You remove snowplow here. I think to be honest we used `*` generically as the content is the same for all package but no worries either way.

Reply:
I kind of had to specify it due to the wording around open source, if we are fine with the current phrasing and plan to update the other models we can of course change it to be universal.